packages/design/src/DataTable/Paged/Pager/Pager.jsx

```diff
@@ -21,17 +21,34 @@ import { Text, Flex } from 'design';
 import PropTypes from 'prop-types';
 
 export default function Pager(props) {
-  const { startFrom = 0, endAt = 0, totalRows = 0, onPrev, onNext } = props;
+  const {
+    startFrom = 0,
+    endAt = 0,
+    totalRows = 0,
+    onPrev,
+    onNext,
+    fetchMore,
```

Comment:
onFetch is more consistent with existing `onPrev` and `onNext` than fetchMore

Reply:
you can use 'fetchMore' in other places but use the `onFetch` prop to pass it to pager

Reply:
revised: https://github.com/gravitational/webapps/pull/329/commits/71d866e09b647e8d05a69a1059e77da7a6b5bd95